Detectors/Upgrades/ALICE3/TRK/simulation/src/Detector.cxx

```diff
@@ -77,10 +77,10 @@ void Detector::configDefault()
   mLayers.clear();
 
   LOGP(warning, "Loading default configuration for ALICE3 TRK");
-  mLayers.emplace_back(0, std::string{GeometryTGeo::getTRKLayerPattern() + std::to_string(0)}, 0.55f, 50.f, 100.e-4); // Adjusted rmin not to overlap with inner BP
+  mLayers.emplace_back(0, std::string{GeometryTGeo::getTRKLayerPattern() + std::to_string(0)}, 0.5f, 50.f, 100.e-4);
   mLayers.emplace_back(1, std::string{GeometryTGeo::getTRKLayerPattern() + std::to_string(1)}, 1.2f, 50.f, 100.e-4);
   mLayers.emplace_back(2, std::string{GeometryTGeo::getTRKLayerPattern() + std::to_string(2)}, 2.5f, 50.f, 100.e-4);
-  mLayers.emplace_back(3, std::string{GeometryTGeo::getTRKLayerPattern() + std::to_string(3)}, 3.78f, 124.f, 100.e-3); // Adjusted rmin not to overlap with outer BP
+  mLayers.emplace_back(3, std::string{GeometryTGeo::getTRKLayerPattern() + std::to_string(3)}, 3.78f, 124.f, 100.e-3);
```

Comment:
If you are sure we do not have overlap anymore then this should be set to official radius as well.
```suggestion
  mLayers.emplace_back(3, std::string{GeometryTGeo::getTRKLayerPattern() + std::to_string(3)}, 3.75f, 124.f, 100.e-3);
```

Reply:
https://github.com/AliceO2Group/AliceO2/blob/dev/Detectors/Upgrades/ALICE3/TRK/simulation/src/Detector.cxx#L83

Here it is 3.78 as well. I can check what happens if you change it to 3.75 to see if there are any overlaps.

Reply:
This does cause an overlap. The vacuum vessel has an inner radius of 3.7 cm and a thickness of 0.08 cm.
3.78 would then be the minimum inner radius for L3 of the TRK in order not to cause an overlap.